src/app/data/forms/form-control.ts

```diff
@@ -21,6 +22,9 @@ export abstract class FormControl extends TitelControl {
   toXml(): string {
     let xml = super.toXml();
     xml += `<ccid>${this.ccid}</ccid>`;
+    if (this.value) {
```

Comment:
Den Wert eines Formularelementes wollen wir nicht in die Formularbeschreibung schreiben. Deswegen muss der Wert hier nicht mitgeschrieben werden, außerdem wäre das dann nicht mehr mit dem xsd kompatibel.